addons/l10n_ar_withholding/models/l10n_ar_earnings_scale.py

```diff
@@ -0,0 +1,47 @@
+from odoo import models, fields
+
+
+class L10nArEarningsScale(models.Model):
+    _name = 'l10n_ar.earnings.scale'
+    _description = 'l10n_ar.earnings.scale'
+
+    name = fields.Char(required=True)
+    line_ids = fields.One2many('l10n_ar.earnings.scale.detail', 'scale_id')
+
+
+class L10nArEarningsScaleDetail(models.Model):
+    _name = 'l10n_ar.earnings.scale.detail'
+    _description = 'l10n_ar.earnings.scale.detail'
+    _order = 'from_amount, to_amount'
+
+    scale_id = fields.Many2one(
+        'l10n_ar.earnings.scale', required=True, ondelete='cascade',
+        help="Calculation of the withholding amount: From the taxable amount (tax base + tax bases applied this month to same tax and partner - non-taxable minimum) subtract the immediately previous amount of the column 'S/ Exced. de $' to detect which row to work with and apply the percentage of said row to the result of the subtraction. Then add to this amount the amount of the '$' column."
+    )
+    currency_id = fields.Many2one(
+        'res.currency', default=lambda self: self.env.ref('base.ARS'),
```

Comment:
store False (compute)?

Reply:
@jco-odoo filled with store=False . I think that I have to refer always to 'base.ARS' , so I didn´t add compute. @vbe-odoo